server/routes/patients/index.js

```diff
@@ -11,5 +11,6 @@ router.get('/', getPatients);
 router.get('/search', getPatientByNameOrPhone);
 router.route('/:patientId').get(getAllProfileData).patch(patchPatientData);
 router.post('/:patientId/history', addHistoryLog);
+router.patch('/:patientId/appointment/:appointmentId');
```

Comment:
I think we already deleted this before and it came again with the conflicts

Reply:
I'll delete it.